scripts/gcloud-sql-export.sh

```diff
@@ -42,8 +42,13 @@ BACKUP_SCHEDULE=${BACKUP_SCHEDULE:-"none"}
 function gcloud_activate_service_account() {
   local file="$1"
 
-  gcloud auth activate-service-account --key-file="$file"
-  gcloud config set project "$(jq -r .project_id "$file")"
+  # Check we have credentials-file and that it exists
+  # We also continue since we'll assume the app could support workload-identity...
+  if  [ -n "${file}" ]  && [ -f "${file}" ] ; then
+    gcloud auth activate-service-account --key-file="$file"
+  fi
+
+  gcloud config set project "${GOOGLE_PROJECT_ID}"
```

Comment:
should we still use the file in the IF condition above since we are activating it anyway ?

Reply:
Hmm not sure what you mean?

Reply:
yeah i was not very clear i know :)

I meant since we still do the ` if  [ -n "${file}" ]  && [ -f "${file}" ] ; then` and handle the key-file when required for testing , should we , in that case, still pull the project from the file and not worry about the Env variable ? 

I don't care that much so i will approve this and let you chose

Reply:
Adding extra note - init gcloud auth via `gcloud auth list` or similar too.